tests/foreman/longrun/test_capsule_loadbalancer.py

```diff
@@ -0,0 +1,315 @@
+"""Test class for Capsule Loadbalancer
+
+:Requirement: Satellite, capsules(3), loadbalancer(haproxy) and client(1)
+
+:CaseAutomation: ManualOnly
+
+:CaseLevel: Integration
+
+:CaseComponent: Capsule Loadbalancer
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+
+from robottelo.decorators import tier1
+from robottelo.test import TestCase
+
+
+class CapsuleLBTestCase(TestCase):
+    """The initial setup:
+        1. Add the manifest into the satellite.
+        2. Make sure the subscriptions are required subscriptions are enabled.
+        3. Sync the repositories.
+        4. Create lifecycle environment, content view and activation key for capsules and client.
+            4.1 Add a puppet module to client(for bootstrap.py).
+        5. Install and register 2 capsules.
+            5.1 Add the respective lifecycle environment to capsule and sync.
+            5.2 Setup the puppetCA and puppet agent amongst the capsules accordingly.
+        6. Create a host group with default values.
+        7. Install loadbalancer(haproxy)
+            7.1 Algorithms as follows:
+                a) :HTTPS - Source
+                b) :HTTP - RoundRobin
+                c) :AMQP - RoundRobin
+                d) :Puppet - RoundRobin
+                e) :RHSM - RoundRobin
+                f) :SCAP - RoundRobin
+            7.2 Enable logging for haproxy (if disabled)
+    """
+
+    @pytest.mark.stubbed
+    @tier1
+    def test_katello_cert_download_manually(self):
+        """Download katello-ca-consumer-latest.noarch.rpm through LB
+
+        :id: 9398d59e-e141-11ea-83a6-4ceb42ab8dbc
+
+        :Steps: run `yum localinstall -y \
+                http://loadbalancer.example.com/pub/katello-ca-consumer-latest.noarch.rpm`
+
+        :expectedresults: Katello cert should be installed from any one of the capsules.
+
+        :CaseLevel: Integration
+
+        :CaseComponent: Installing
+        """
+        pass
+
+    @pytest.mark.stubbed
+    @tier1
+    def test_manually_register_client_using_ak(self):
+        """Register the client using ak to the capsule
+
+        :id: 7318c380-e149-11ea-9b17-4ceb42ab8dbc
+
+        :Steps: run `subscription-manager register --org=Your_Organization \
+                --activationkey=Your_Activation_Key \
+                --serverurl=https://loadbalancer.example.com:8443/rhsm \
+                --baseurl=https://loadbalancer.example.com/pulp/repos`
+
+        :expectedresults: The client should be registered throught one of the capsules
+
+        :CaseLevel: Integration
+
+        :CaseComponent: Registration
+        """
+        pass
+
+    @pytest.mark.stubbed
+    @tier1
+    def test_list_repolist(self):
+        """List all the repositories
```

Comment:
Well, all these tests are to be run manually only, so I don't think it would matter the order of the execution. When the test( say X ) is run manually, the previous tests need to be successful in order to pass the current (X) test.

Reply:
Ah, OK. I thought you commit them now with intent of automating them later. If you don't plan to automate them anytime soon, then my comment is not really relevant. Sorry for noise.